packages/react-core/src/components/Dropdown/DropdownItem.tsx

```diff
@@ -40,7 +40,7 @@ export interface DropdownItemProps extends InternalDropdownItemProps, OUIAProps
   /** Custom item rendering that receives the DropdownContext */
   customChild?: React.ReactNode;
   /** tabIndex to use, null to unset it */
```

Comment:
Should this description be updated now that `null` is no longer an acceptable value?

Reply:
@thatblindgeye the functionality is still the same correct? The prop will still take a null value, so a null value can still be set to unset it. 
Maybe we can add unit  for a sanity check.

Reply:
I don't get any errors when trying to pass `tabIndex={null}` in an example file. For a unit test, would we just want to check that passing in null results in no `tabindex` attribute being applied?

Reply:
@thatblindgeye yes.

Reply:
Sounds good - sorry for the misunderstanding!

Reply:
If it used to accept null, and now it's going to throw an error, is that a breaking change?

Reply:
Titani did mention it would be. My recent commit just now adds back `null` types to the TreeviewListItem and DropdownItem components, and adds in an `Omit...` to both to resolve the TS error being flagged in strict mode